README.md

```diff
@@ -1 +1,7 @@
-# stencyl-flatpak
\ No newline at end of file
+# Flatpak for Stencyl
+
+http://www.stencyl.com/
+
+## TODO:
+- Switch to a non endless runtime
```

Comment:
Could you write what would need to be done for this to happen? What would need to be taken from the endless sdk/runtime that does not exist in f.d.o. or GNOME runtimes? Like add java, 32bit libraries and so on.